src/api/views.py

```diff
@@ -55,4 +71,16 @@ class TodoViewSet(viewsets.ModelViewSet):
     def perform_create(self, serializer):
         user = self.request.user
         creator = user if user.is_authenticated else None
-        serializer.save(creator=creator)
\ No newline at end of file
+        serializer.save(creator=creator)
+
+
+def get_readiness_status(request):
+    if time.time() < start_time + startup_period:
+        return HttpResponse("Not Ready", status=503)
+    else:
+        return HttpResponse("Ready", status=200)
+
+
+def get_healthy_status(request):
+    return HttpResponse("Healthy", status=200)
+    
```

Comment:
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_3_write_a_manifest/assets/87720583/e6b944e2-42a6-4239-9ae0-e65990eb6df1)
EoF doesn't  work because last line isn't empty 
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_3_write_a_manifest/assets/87720583/f8cd69f0-e24e-4d63-b293-e0a3fbdb395f)